### ISSUE TITLE
Accessibility - WCAG 2.1 Level A and AA - Bypass Block - Set keyboard focus on the main parts of each page to fulfill the criterion

### ISSUE THREAD

**Issue by davidverdu** [IQSS/ondemand-loop#169]
Ensure WCAG 2.1 Level AA **2.4.1 Bypass Block** criterion in the application by **programmatically set the keyboard focus on the most valuable item of each page** when the user enters a page, so that the user doesn't have to navigate across all menu items links over and over again when they enter a new page.

- https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html

## Tools
- [WAVE](https://wave.webaim.org/extension/)
- [axe DevTools](https://www.deque.com/axe/) 
- [More tools](https://accessibility.huit.harvard.edu/auto-tools-testing#free)

## Reference
- [Harvard Accessibility techniques](https://accessibility.huit.harvard.edu/techniques)
- [Evaluating Web Accessibility overview](https://www.w3.org/WAI/test-evaluate/)
- [WCAG 2.1 Guidelines](https://www.w3.org/TR/WCAG21/)

**Comment by davidverdu** [comment]
I asked how to implement this criterion at the Accessibility Training and the best practice is, for pages with a lot of navigation options like in Harvard, **provide a link at the top called "Go to the Main part of the page" to bypass the navigation links**. The usage of `tabindex` with values greater than 0 is not recommended. So, the best way to implement it is to provide the bypass link. 

However, Loop application has only a few navigation options and is not a public web, but an application for a niche of cluster users. This is why I consider it is not worth implementing the bypass link on this application.

**Comment by abujeda** [comment]
Implementation reviewed and merged
